valhalla/baldr/graphtileheader.h

```diff
@@ -601,13 +603,13 @@ class GraphTileHeader {
 
   // TODO: in v4, don't store this its superfluous information, the graphid has all we need
   // Base lon, lat of the tile
-  std::pair<float, float> base_ll_;
+  std::pair<float, float> base_ll_ = {0, 0};
 
   // baldr version.
-  char version_[kMaxVersionSize];
+  std::array<char, kMaxVersionSize> version_ = {};
```

Comment:
similar to admin id prefer not to put `std::array` in here (base_ll_ is also marked for deletion)